prow/spyglass/lenses/podinfo/template.html

```diff
@@ -6,6 +6,7 @@
 
 {{define "body"}}
 {{$pod:=.PodReport.Pod}}
+{{$podLink:=.PodReport.PodLink}}
```

Comment:
I think this should just be `.PodLink` without the `.PodReport` prefix. Can we have add a unit test that uses the real template rather than mocking the template?